src/site/stages/build/process-cms-exports/transformers/node-leadership_listing.js

```diff
@@ -17,6 +18,32 @@ const transform = entity => ({
   fieldLeadership: entity.fieldLeadership[0],
   fieldMetaTitle: getDrupalValue(entity.fieldMetaTitle),
   fieldOffice: entity.fieldOffice[0],
+  reverseFieldListingNode: {
+    entities: entity.reverseFieldList
+      ? entity.reverseFieldList
+          .filter(
+            reverseField =>
+              reverseField.entityBundle === 'person_profile' &&
+              reverseField.isPublished,
+          )
+          .map(reverseField => ({
+            title: reverseField.title,
+            fieldNameFirst: reverseField.fieldNameFirst,
+            fieldLastName: reverseField.fieldLastName,
+            fieldSuffix: reverseField.fieldSuffix,
+            fieldDescription: reverseField.fieldDescription,
+            fieldOffice: reverseField.fieldOffice,
+            fieldIntroText: reverseField.fieldIntroText,
+            fieldPhotoAllowHiresDownload:
+              reverseField.fieldPhotoAllowHiresDownload,
+            fieldBody: reverseField.fieldBody,
+            changed: reverseField.changed,
+            entityUrl: reverseField.entityUrl,
+            fieldMedia: reverseField.fieldMedia,
+          }))
+          .sort((a, b) => b.created - a.created)
```

Comment:
I don't see `created` in the output schema for `node-person_profile`, nor is it in the above object literal. Did you forget to add that to the `node-person_profile` bundle, or did you mean to sort by `changed`?